portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles user logins. */
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    PrintWriter out = response.getWriter();
+    out.println("<h1>Login Status</h1>");
+
+    // Only logged-in users can see the form
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      String logoutUrl = userService.createLogoutURL("/login");
```

Comment:
should be /logout?

Reply:
It's the same url for login and logout because the components change depending on if you're logged in or not. In that case, I'll just declare one url var instead of 2.